File: ruby/tests/601-show-user.rb
admin   = admin_handle

raise "SKIPPED: no Users" unless @user_list_600.present?

user_id   = @user_list_600.first[:id]
user_info = admin.show_user(user_id)

puts "USER_INFO: #{user_info.inspect}" if @debug

raise "Value returned is not an hash" unless user_info.present? && user_info.is_a?(Hash)
raise "You have no information for your user" if user_info.empty?

true

File: perl/tests/601-show-user.pl
my $admin = &AdminHandle;
our $user_list_600;

die "SKIPPED: no Users\n" if scalar(@$user_list_600) == 0;

my $user_id   = $user_list_600->[0]->{'id'};
my $user_info = $admin->show_user($user_id);

if ($DEBUG) {
  my $dump_user_info = Data::Dumper::DumperX($user_info);
  print "USER_INFO: $dump_user_info\n";
}

die "Value returned not an hash.\n" unless defined($user_info) && ref($user_info) eq "HASH";
die "You have no information for your user\n" if scalar(%$user_info) eq "0";

1;
